src/Apps/Auction/Components/__stories__/PricingTransparency.story.tsx

```diff
@@ -6,7 +6,23 @@ import { PricingTransparency } from "../PricingTransparency"
 storiesOf("Apps/Auction/Components", module).add("Pricing Transparency", () => {
   return (
     <Section>
-      <PricingTransparency />
+      <PricingTransparency
+        artwork={{
+          saleArtwork: {
+            calculatedCost: {
+              bidAmount: {
+                display: "$5,000",
+              },
+              buyersPremium: {
+                display: "$1,000",
+              },
+              subtotal: {
+                display: "$6,000",
+              },
+            },
+          },
+        }}
+      />
```

Comment:
@erikdstock we might want to update this storybook to use a query renderer as well, but it's been too long since I touched storybooks in Reaction. Let me know if you want to pair on this.